Docs/user_guide/getting_started.md

```diff
@@ -6,29 +6,35 @@ Welcome to the 3D Slicer community. This page contains information that you need
 
 3D Slicer runs on any Windows, Mac, or Linux computer that was released in the last 5 years. Older computers may work (depending mainly on graphics capabilites). Slicer can also run on virtual machines and docker containers.
 
+## Operating system versions
+
+|               |Windows                                        |macOS                             |Linux                                     |
+|---------------|-----------------------------------------------|----------------------------------|------------------------------------------|
+|Stable Release |Windows 10 or later<br>Windows 8.1<br>Windows 7|OS X Mavericks (10.9) or later    |Ubuntu 16.04 or later<br>CentOS 7 or later|
+|Preview Release|Windows 10 or later<br>Windows 8.1<br>Windows 7|macOS High Sierra (10.13) or later|Ubuntu 18.04 or later<br>CentOS 7 or later|
+
+**Notes:**
+- It is always recommended to use the latest public release of your operating system.
+- Most recent versions of popular Linux distributions should work.
+- Only 64-bit Slicer versions are available to download. Developers can attempt to build 32-bit versions on their own if they need to run Slicer on a 32-bit operating system.
```

Comment:
I like the previous text associated with `32 bit versus 64 bit`, may be we could keep it. Or even introduce a `Frequently Asked Question` sub-section ?

Reply:
Yeah the previous text seemed to have been written in a time where 32-bit was more common than 64-bit or at least closer to half and half, so I was attempting to re-write it in the manner of expecting 64-bit with the option to choose 32-bit.  Even when running the suggested latest version of macOS it only supports running 64-bit applications so it gets a little more tricky of a comment as 32-bit support gets less and less supported.

Reply:
Good point.

What about something like this then:

```
- Only 64-bit Slicer installers are available to download. Developers can attempt to build 32-bit versions on their own if they need to run Slicer on a 32-bit operating system.  That said, this should be carefully considered as many clinical research tasks, such as processing of large CT or MR volumetric datasets, require more memory require more memory than can be accommodated with a 32 bit program.
```

Reply:
Yes, that sounds good.  I've updated the commit with this change.